### ISSUE TITLE
question about saving model as SavedModel

### ISSUE THREAD

**Issue by yuanhuang0809** [GeneZC/Chinese-NER#5]
Hi,
I am trying to save the model as SavedModel. Could you plase help me with defining about inputs and outputs?

I think the model's inputs and outputs should be like this:
inputs={"ChatInputs": model.char_inputs, "Dropout": model.dropout},
outputs={"project/Reshape": model.logits, "crf_loss/transitions":model.trans}

Is this right?
Thank you!
